Course_Part_10/src/TicTacToeBasic.java

```diff
@@ -0,0 +1,247 @@
+import java.sql.Array;
+import java.util.*;
+
+
+public class TicTacToeBasic {
+
+    static TreeMap<String, String> playOptions;
```

Comment:
waarom heb je gekozen voor een TreeMap ipv een HashMap?

Reply:
https://www.javatpoint.com/difference-between-hashmap-and-treemap
Basis                      | HashMap                                                 | TreeMap
Order of elements | HashMap does not maintain any order. | The elements are sorted in natural order (ascending).

Reply:
yes, maar heb je t ook nodig ? 

> Performance | HashMap is faster than TreeMap because it provides constant-time performance that is O(1) for the basic operations like get() and put(). | TreeMap is slow in comparison to HashMap because it provides the performance of O(log(n)) for most operations like add(), remove() and contains().
> -- | -- | --
>

Reply:
Ik weet waar de verwarring vandaan komt.
Voor displayBoard() wou ik loopen over de Map, playOptions.forEach((key, value) en de logica die ik toen wou gebruiken had een ordered Map nodig. Maar ik heb het uiteindelijk anders geschreven. Je hebt dus gelijk is niet meer relevant.